### PR TITLE
[2.5] PANDARIA: update gpu monitoring to v2

### PR DESCRIPTION
**Pull request by jianghang8421**
https://github.com/cnrancher/pandaria/issues/1499

### CODE REVIEW

File: charts/rancher-monitoring/v0.2.1003/charts/exporter-gpu-node-v1/templates/daemonset.yaml [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+              mountPath: /run/prometheus
+        - image: "{{ template "system_default_registry" . }}{{ .Values.image.dcgmexporter.repository }}:{{ .Values.image.dcgmexporter.tag }}"
+          name: nvidia-dcgm-exporter
+          securityContext:
+            runAsNonRoot: false
+            runAsUser: 0
+          volumeMounts:
+            - name: collector-textfiles
+              mountPath: /run/prometheus
+
```
Comment (Jason-ZW) [inline charts/rancher-monitoring/v0.2.1003/charts/exporter-gpu-node-v1/templates/daemonset.yaml]: useless line

File: charts/rancher-monitoring/v0.2.1003/charts/exporter-gpu-node-v1/templates/daemonset.yaml [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+      hostNetwork: true
+      hostPID: true
+
```
Comment (Jason-ZW) [inline charts/rancher-monitoring/v0.2.1003/charts/exporter-gpu-node-v1/templates/daemonset.yaml]: useless line

File: charts/rancher-monitoring/v0.2.1003/values.yaml [hunk 3]
```diff
@@ -417,3 +417,5 @@ exporter-gpu-node:
     dcgmexporter:
       repository: cnrancher/dcgm-exporter
       tag: 1.4.6
```
Comment (Jason-ZW) [inline charts/rancher-monitoring/v0.2.1003/values.yaml:419]: just confirm 1.4.6 is correct?

Comment (jianghang8421) [reply charts/rancher-monitoring/v0.2.1003/values.yaml:419]: 1.4.6 is gpu monitoring v1 image version

**Review by Jason-ZW — APPROVED** [review]
LGTM

**Pull request merged**